INN/umbrella-publicsource
# Giving tuesday updates

## Changes

This pull request makes the following changes:

- Adds category header widget area functionality like we did for MWCIR at https://github.com/INN/umbrella-mwcir/pull/43
- Shows the donate button in the header on mobile
- Updates styling for Largo donate + subscribe button in the footer
![Screen Shot 2020-05-04 at 3 44 22 PM](https://user-images.githubusercontent.com/18353636/81006842-28a0a080-8e1e-11ea-81f5-ded89b304131.png)

## Why



For #39

## Testing/Questions

Features that this PR affects:

- Donate button in mobile menu
- Category archive widget area in header
- Footer widget styling


Questions that need to be answered before merging:


Steps to test this PR:

Donate button:
1. View the site on mobile and verify the donate button appears in the header

Category header widget area:
1. Test the category archive widget area the same way we did for MWCIR https://github.com/INN/umbrella-mwcir/pull/43

Footer widgets:
1. Add a Largo Donate widget to the Footer 2 widget area
2. Add an HTML widget under the Donate widget with this text
```
<a class="btn btn-subscribe" href="/subscribe/">Subscribe</a>
```
3. View on the frontend and verify it looks ok on desktop and mobile
3.
